cunumeric/runtime.py

```diff
@@ -169,7 +170,7 @@ def destroy(self) -> None:
         assert not self.destroyed
         if self.num_gpus > 0:
             self._unload_cudalibs()
-        if hasattr(self, "args") and settings.report_coverage():
+        if settings.report_coverage():
```

Comment:
`args` is gone now so this `hasattr` always failed and so coverage reporting would never be activated.